Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM mcr.microsoft.com/vscode/devcontainers/rust:0-1
-
+FROM rust:1.64 as base
```

Comment:
Can also use this base: `FROM paritytech/ci-linux:production as builder` which is used for most parity/substrate projects

if it works how it is i would leave it for now though

Reply:
@Romulus10 lets make a notion ticket for this?

Reply:
Absolutely. Additionally this current conf is an experiment at build optimization. Feel free to suggest improvements if you find problems.